ramls/inventory.raml

```diff
@@ -76,7 +77,30 @@ resourceTypes:
               body:
                 text/plain:
                   example: "Internal server error"
-
+    /move:
```

Comment:
does it make sense to have a separate move API? (i.e. inventory-move.raml)?

Reply:
It's OK for me. Let's move this endpoint to separate raml API. +  we will add the same for holdings in the nearest future.

Reply:
Please make the version of this new interface 0.1 to mark it as experimental

Reply:
> does it make sense to have a separate move API? (i.e. inventory-move.raml)?

Maybe give it the name `inventory-move-items` for moving items between holdings, to distinguish it from moving holdings between instances

Reply:
@marcjohnson-kint let it be more general inventory-move.raml. I think it will be good to have items and holdings move in the one file.

Reply:
Ok, that means that any potential implementor would need to implement both.